torch/cuda/__init__.py

```diff
@@ -67,7 +67,7 @@ def _check_capability():
     old_gpu_warn = """
     Found GPU%d %s which is of cuda capability %d.%d.
     PyTorch no longer supports this GPU because it is too old.
-    The minimum cuda capability that we support is 3.5.
+    The minimum cuda capability that we support is 3.0.
```

Comment:
Can we dynamically extract minimum supported version from `torch.cuda.get_arch_list()`?